pkg/kv/kvserver/replica_app_batch.go

```diff
@@ -288,6 +288,13 @@ func (b *replicaAppBatch) runPostAddTriggersReplicaOnly(
 		}
 		res.AddSSTable = nil
 	}
+	if res.LinkExternalSSTable != nil {
+		if res.LinkExternalSSTable.RemoteRewriteTimestamp.IsSet() {
```

Comment:
What's this timestamp check for?

Reply:
i was copying the AddSSTable req code above. I'll copy the comment: 
```
// We've ingested the SST already (via the appBatch), so all that's left
// to do here is notify the rangefeed, if appropriate.
```

Reply:
And we check the timestamp bc non mvcc ExternalSSTable requests are not sent to rangefeeds.